constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -197,12 +206,9 @@ func (d *Driver) Query(ctx context.Context, target string, constraints []*unstru
 			VersionedOldObject: request.GetOldObject(),
 			VersionedObject:    request.GetObject(),
 		}
-		parameters, _, err := unstructured.NestedMap(constraint.Object, "spec", "parameters")
-		if err != nil {
-			return nil, err
-		}
-		ctx := context.WithValue(ctx, variablesKey, map[string]interface{}{"params": parameters})
-		response := validator.Validate(ctx, versionedAttr, constraint, celAPI.PerCallLimit)
+
+		// TODO: should namespace be made available, if possible? Generally that context should be present
```

Comment:
should we open an issue for this to make sure we add ns later?

Reply:
Maybe? Since this is still prototype code and will need to be re-worked for testing etc., was keeping things lightweight.

Reply:
LMK if you definitely want to see an issue.

Reply:
it’s just a suggestion, not a PR blocker